easybuild/easyconfigs/b/Bader/Bader-1.04-intel-2020b.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Bader'
+version = '1.04'
+
+homepage = 'http://theory.cm.utexas.edu/henkelman/code/bader/'
+description = "A fast algorithm for doing Bader's analysis on a charge density grid."
+
+toolchain = {'name': 'intel', 'version': '2020b'}
```

Since I this doesn't use any MPI or MKL, this can be dropped to `iccifort` level (equivalent to GCC)
```suggestion
toolchain = {'name': 'iccifort', 'version': '2020.4.304'}
```

---

the filename should be changed to match the toolchain change as well